packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -53,14 +54,26 @@ export default class Editor {
      * @param contentDiv The DIV HTML element which will be the container element of editor
      * @param options An optional options object to customize the editor
      */
-    constructor(contentDiv: HTMLDivElement, options: EditorOptions = {}) {
+    constructor(contentDiv: HTMLDivElement, options: EditorOptions = {}, darkMode: boolean | DarkModeOptions = false) {
         // 1. Make sure all parameters are valid
         if (getTagOfNode(contentDiv) != 'DIV') {
             throw new Error('contentDiv must be an HTML DIV element');
         }
 
+        // Set up dark mode defaults if not provided.
+        if (darkMode && typeof darkMode === 'boolean') {
+            darkMode = <DarkModeOptions>{
+                defaultFormat: <DefaultFormat>{

```

Comment:
Is there a better place to store this formatting? In a const, perhaps in a separate file?

Reply:
+1 for separate file